mintpy/objects/coord.py

```diff
@@ -450,26 +450,6 @@ def check_box_within_data_coverage(self, pixel_box, print_msg=True):
             msg += '\tsubset range in (W, N, E, S): {}\n'.format(self.box_pixel2geo(pixel_box))
             raise ValueError(msg)
 
-        # Check Y/Azimuth/Latitude subset range
-        if sub_y[0] < 0:
-            sub_y[0] = 0
-            if print_msg:
-                print('WARNING: input y < min (0)! Set it to min.')
-        if sub_y[1] > length:
-            sub_y[1] = length
-            if print_msg:
-                print('WARNING: input y > max ({})! Set it to max.'.format(length))
-
-        # Check X/Range/Longitude subset range
-        if sub_x[0] < 0:
-            sub_x[0] = 0
-            if print_msg:
-                print('WARNING: input x < min (0)! Set it to min.')
-        if sub_x[1] > width:
-            sub_x[1] = width
-            if print_msg:
-                print('WARNING: input x > max ({})! Set it to max.'.format(width))
-
         out_box = (sub_x[0], sub_y[0], sub_x[1], sub_y[1])
```

Comment:
`check_box_within_data_coverage()` is used in other places besides view.py. For the purpose described in this PR, we should just not calling this function inside view.py.

Reply:
Done, this call is removed from view.py

Reply:
Let's also do not modify this function also so that other scripts won't be affected.

Reply:
@yunjunz ok, went ahead and tweaked this accordingly. Let me know what you think.